src/main/scss/templates/_friends.scss

```diff
@@ -0,0 +1,53 @@
+// Friends styling
+.friends {
+  display: block;
+  padding: $size-base * 2;
+
+  .card {
+    background-color: $color-white;
+    padding: $size-base * 2;
+    border-radius: $size-base;
+    margin-right: $size-base * 3;
+    margin-bottom: $size-base * 3;
+    text-align: center;
+    width: 250px;
+    height: 300px;
+
+    .username {
+      font-size: $size-base * 2;
+      font-weight: bold;
+    }
+
+    .user-type {
+      font-size: $size-base * 1.5;
+    }
+
+    .profile-container {
+      height: 200px;
+      line-height: 200px;
+      margin: $size-base * 1.5;
+    }
+
+    .profile-photo {
```

Comment:
I dunno how I feel about the differently sized photos. Consider unifying the sizes or forcing them to all be circular (even if they're rectangular in aspect ratio) and adding a border (much like the sidebar photo).

![example](http://i.imgur.com/sdZlosB.png)
Feels a little messy if they're like this.